ui/pages/notifications/notification-components/swap-completed/swap-completed.tsx

```diff
@@ -141,7 +153,7 @@ export const components: NotificationComponent<SwapCompletedNotification> = {
                 position: BadgeWrapperPosition.topRight,
               },
             }}
-            label={t('notificationItemSwapped') || ''}
+            label={t('notificationItemTo') || ''}
```

Comment:
Good bug fix. This second field should show the "To" text.

<details><summary>See Figma screenshot</summary>

![Screenshot 2024-06-05 at 10 40 54](https://github.com/MetaMask/metamask-extension/assets/17534261/5bd6d6f2-fa1a-4b20-b964-f8abeda0e2e2)
</details>